Packs/DeveloperTools/Scripts/WaitAndCompleteTask/WaitAndCompleteTask.py

```diff
@@ -0,0 +1,186 @@
+from time import sleep
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+from typing import Dict, Any
+import traceback
+
+POSSIBLE_STATES = ['New', 'InProgress', 'Completed', 'Waiting', 'Error', 'Skipped', 'Blocked']
+
+''' STANDALONE FUNCTION '''
+
+
+def get_incident_tasks_by_state(incident_id: int, task_states: Optional[list] = None) -> list:
+    """
+
+    Args:
+        incident_id: Incident id to pull its tasks.
+        task_states: States to get. None to get all tasks with all tasks.
+
+    Returns:
+        Tasks with given states related to given incident.
+    """
+    args: Dict[str, Any] = {
+        'incidentId': incident_id
+    }
+    # leave states empty to get all tasks
+    if task_states:
+        args['states'] = ','.join(task_states)
+    try:
+        raw_response = demisto.executeCommand('DemistoGetIncidentTasksByState', args=args)
+        return raw_response[0].get("Contents") if raw_response[0].get("Contents") else []
+    except Exception as e:
```

Comment:
errors won't raise as exceptions, please use is_error() and get_error()
fix only that (also in complete_task_by_id()), great job 🙏🏽